components/datadog/apps/nginx/ecsFargate.go

```diff
@@ -29,12 +28,12 @@ func FargateAppDefinition(e aws.Environment, clusterArn pulumi.StringInput, apiK
 	opts = append(opts, pulumi.Parent(EcsFargateComponent))
 
 	alb, err := lb.NewApplicationLoadBalancer(e.Ctx, namer.ResourceName("lb"), &lb.ApplicationLoadBalancerArgs{
-		Name:           e.CommonNamer.DisplayName(32, pulumi.String(appName)),
+		Name:           e.CommonNamer.DisplayName(32, pulumi.String("nginx-fg")),
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Why not keeping `appName` and using it only here and for the `TargetGroup` ?